src/nacl/signing.py

```diff
@@ -59,7 +61,20 @@ class VerifyKey(encoding.Encodable, StringFixer, object):
     :param encoder: A class that is able to decode the `key`
     """
 
-    def __init__(self, key, encoder=encoding.RawEncoder):
+    def __init__(self, key, *args, **kwargs):
+        if len(args) == 1:
+            encoder = args[0]
+        elif len(args) > 1:
+            raise TypeError("Too many arguments")
+        else:
+            encoder = kwargs.get('encoder', encoding.RawEncoder)
+
+        if 'encoder' in kwargs or len(args) > 0:
+            warn(("Implicit encoding/decoding of signing keys "
+                  "is deprecated and will get removed in a future release. "
```

Comment:
s/will get/will be

Reply:
Fixed by just pushed 7652934908e665625dee8797d274e30d0ee6dae3